src/components/Task/TaskRow/CommentTooltipText.tsx

```diff
@@ -0,0 +1,30 @@
+import { Typography } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { DateTime } from 'luxon';
+import { useLocale } from 'src/hooks/useLocale';
+import { dateFormat } from 'src/lib/intlFormat';
+
+export const TooltipTypography = styled(Typography)(() => ({
+  fontSize: 11,
+}));
+
+interface CommentTooltipTextProps {
+  comments: { id: string; body: string; updatedAt: string }[];
+}
+export const CommentTooltipText: React.FC<CommentTooltipTextProps> = ({
+  comments,
+}) => {
+  const locale = useLocale();
+
+  const latestComment = !!comments.length
+    ? comments[comments.length - 1]
+    : null;
```

Comment:
`.at(index)` is similar to `[index]` but it can also take negative numbers to index backwards from the end of the array, i.e. `array.at(-1)` is the last element, `array.at(-2)` is the second-to-last element, etc.

```suggestion
  const latestComment = comments.length.at(-1) ?? null;
```